### PR TITLE
Add multi-input example

### LINKED ISSUE #13: Example for multiple inputs
We should provide an example of using multiple inputs to the model.

Is there a pre-existing trained net we can deploy?
Davenet?

Do this after restructuring examples for #12 

Also a test?

### PR DESCRIPTION
**Pull request by jatkinson1000**
Closes #13 

- [x] Add Python components of WaveNet
- [ ] Add fortran inference code
- [ ] Ensure README and build instructions complete

### CODE REVIEW

File: examples/3_Multiple_Inputs/pt2ts.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    # -----------
+    script_to_torchscript(trained_model, filename=saved_ts_filename)
+
+    # -----------
+    # Tracing
+    # -----------
+    # trace_to_torchscript(trained_model, trained_model_dummy_input, filename=saved_ts_filename)
+
+    # Load torchscript and run model as a test
+    testing_input_u = 2.0 * trained_model_dummy_input_u
```
Comment (surbhigoel77) [inline examples/3_Multiple_Inputs/pt2ts.py:120]: Maybe add a line to why the input is scaled by a factor of 2

File: examples/3_Multiple_Inputs/pt2ts.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
+"""Load a pytorch model and convert it to TorchScript."""
```
Comment (surbhigoel77) [inline examples/3_Multiple_Inputs/pt2ts.py:1]: You could mention the methods included for torchscript conversion here (tracing and scripting), and that `scripting` is the recommended method

File: examples/3_Multiple_Inputs/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+torch
```
Comment (surbhigoel77) [inline examples/3_Multiple_Inputs/requirements.txt:1]: Adding minimum required versions would be good?

File: examples/3_Multiple_Inputs/run_wavenet.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    model : nn.Module
+        WaveNet model ready to be deployed.
+    wind :
+        U or V (128, num_col, 40)
+    lat :
+        latitudes (num_col)
+    p_surf :
+        surface pressure (128, num_col)
+    Y_out :
+        output prellocated in MiMA (128, num_col, 40)
```
Comment (surbhigoel77) [inline examples/3_Multiple_Inputs/run_wavenet.py:52]: spelling of prelocated

**Review by surbhigoel77 — APPROVED** [review]
LGTM

**Comment by jatkinson1000** [comment]
I am closing this as we now have other examples (e.g. MultiIO) in the examples suite that demonstrate this property, and this was using the old API anyway.

**Pull request closed without merging**